packages/client/wallets/smart-wallet/src/error/index.ts

```diff
@@ -17,24 +14,12 @@ export class SmartWalletSDKError extends Error {
     }
 }
 
-export class NotAuthorizedError extends SmartWalletSDKError {
-    constructor(message: string) {
-        super(message, SmartWalletErrors.NOT_AUTHORIZED);
-    }
-}
-
 export class TransferError extends SmartWalletSDKError {
     constructor(message: string) {
         super(message, SmartWalletErrors.TRANSFER);
     }
 }
 
-export class TransactionError extends SmartWalletSDKError {
```

Comment:
If there's a error during a transaction or signing, we want to throw either the corresponding viem error, or an error related to the signer type (For ex. we'll catch and format webauthn errors w/ better messages for passkeys)